config/helm/kustomization.yaml

```diff
@@ -71,6 +71,12 @@ patches:
     patch: |-
       - op: remove
         path: /spec/template/spec/securityContext/seccompProfile
+  - target:
+      kind: CustomResourceDefinition
+      name: (ipaddressclaims\.ipam|extensionconfigs\.runtime|ipaddresses\.ipam).cluster.x-k8s.io
```

Comment:
label selector would also [work](https://github.com/giantswarm/cluster-api-ipam-provider-in-cluster-app/pull/4/files#diff-12bdb52403497f6a3f9207ddcab47594256ccb9059c755e337c827f8a0742a9dR31)